script.js

```diff
@@ -10,6 +10,7 @@ const courses = ({ title, author, image, price, bestseller }) => {
 
 const newCourses = async (search) => {
   const coursesList = document.getElementsByClassName("coursesList")[0];
+  coursesList.innerHTML = "";

```

Comment:
why you added this line ?

Reply:
when i'm searching for a course and find it he add it to the existing all courses so I'm using "coursesList.innerHTML = " to clear all courses to add the course I'm searching for it.